packages/page-accounts/src/CreateAccount.spec.tsx

```diff
@@ -5,10 +5,31 @@ import AccountsApp from '@polkadot/app-accounts';
 import { MemoryStore } from '@polkadot/app-accounts/test-support/MemoryStore';
 import { Api } from '@polkadot/react-api';
 import '@polkadot/react-components/i18n';
+import { ThemeDef } from '@polkadot/react-components/types';
 import { useApi } from '@polkadot/react-hooks';
-import { fireEvent, render } from '@testing-library/react';
+import { fireEvent, render, waitForElementToBeRemoved } from '@testing-library/react';
 import React, { PropsWithChildren } from 'react';
 import { MemoryRouter } from 'react-router-dom';
+import { ThemeProvider } from 'styled-components';
+
+// copied from packages/apps/src/themes.ts, as we cannot reference apps module
```

Comment:
Single this is a test, we can just pull the import. (And it is in account already)